pkg/controller/v1alpha2/trial/trial_controller.go

```diff
@@ -224,24 +234,20 @@ func (r *ReconcileTrial) reconcileJob(instance *trialsv1alpha2.Trial, desiredJob
 	err = r.Get(context.TODO(), types.NamespacedName{Name: desiredJob.GetName(), Namespace: desiredJob.GetNamespace()}, deployedJob)
 	if err != nil {
 		if errors.IsNotFound(err) {
-			logger.Info("Creating Job", "kind", kind)
+			logger.Info("Creating Job", "kind", kind,
+				"name", desiredJob.GetName())
 			err = r.Create(context.TODO(), desiredJob)
 			if err != nil {
 				logger.Error(err, "Create job error")
 				return nil, err
 			}
+			deployedJob = desiredJob
```

Comment:
Is this needed?  deployed job is returned if job is already in running state. In reconcile function(https://github.com/kubeflow/katib/pull/532/files#diff-63bd3a953ae0c850443cefebf8365018R21), update status if deployed job is not null

Reply:
If we do not set it, deployedJob is an empty job without status. And we have to update it in the next reconcile loop. I think we can do it in this loop.

Reply:
WDYT, actually we keep an eventually consistent model, thus it is not a critical change.

Reply:
Let's remove it since I think it is not a necessary change.

Reply:
OK. agree